src/index.js

```diff
@@ -27,8 +27,22 @@ function syncEvent(node, eventName, newEventHandler) {
 }
 
 export default function (CustomElement, opts) {
+  // Allows to pass the tagName instead of the CustomElement Class
+  // Avoids creating an instance of the component just to get the tagName.
+  // Makes it possible to use this with Web.Components written in TypeScript.
+  var tagName;
+  if (typeof CustomElement === 'string') {
+    tagName = CustomElement;
+    CustomElement = customElements.get(tagName);
```

Comment:
I'm probably missing something, but how does this work? Where is `customElements.get` defined?